### ISSUE TITLE
Effect not showing up.

### ISSUE THREAD

**Issue by AlejandroArciniegas** [MightyPirates/Scannable#7]
The effect is not showing up. 
As it is shown here: 
https://youtu.be/rHHnIL3wajY

Seems to be a conflict with Optifine.

**Comment by fnuecke** [comment]
Please try the latest version (1.2.2), at least for the reporter of #5 that helped.

**Comment by AlejandroArciniegas** [comment]
I tried it, still wont show up the effect. Take a look: https://youtu.be/rHHnIL3wajY

**Comment by AlejandroArciniegas** [comment]
And another quick question, Which mod do I need to enable power in OC for 1.11.2 havn't found any one from the list

**Comment by fnuecke** [comment]
Mm, that log is not the interesting one though, mind putting your `fml-client-latest.log` on gist or pastebin or so after using the scanner? Thanks.

OC 1.11 power is currently not implemented, will be done... this week-ish. Probably.

**Comment by AlejandroArciniegas** [comment]
[fml-client-latest.zip](https://github.com/MightyPirates/Scannable/files/985899/fml-client-latest.zip)
Pastebin says it's to big

**Comment by fnuecke** [comment]
Hmm, it doesn't say anything about failing rendering, so that's... odd. Have you tried setting the config option to not inject the depth texture manually? (`injectDepthTexture = false`)

**Comment by AlejandroArciniegas** [comment]
I just tried it, no results. Also tried with only forge and optifine on their lastest versions and still won't show up the effect then I tried without any other mod than forge and it worked, seems to be related to Optifine and its lastest version. Maybe one of its confirguration options has something disabling the effect? If that wich one could it be? I'd love to have both mods working as my pc is not beefy and optifine helps me to keep 60+ fps but the effect of this mod is quite cool also and would love to see it in a helmet or a glasses in some other tech mod from you. Greetings.    

Update: After a bit of testing I solved it, the "Fast Renderer" option somehow disables the effect. Thanks anyway and keep up your great work!
